proposals/new-project-addition-process.md

```diff
@@ -0,0 +1,36 @@
+## Motivation
```

Comment:
I think, this proposal should also include the following:
- Why this process is needed for Kubeflow community ?
- Explain the responsibilities for project owners.
- How this affect the existing Kubeflow governance structure (Working Groups, KSC, etc.).
- Level of maturity for projects to be donated to the Kubeflow. For example, how many production users, contributors, releases, etc.